svc-api/main.go

```diff
@@ -48,6 +48,21 @@ func main() {
 		}
 		basicAuth := r.Header.Get("Authorization")
 		var basicAuthToken string
+		// This if block added to fix issue odim-98
+		if basicAuth != "" {
```

Comment:
same if condition check is happening in line number 66 also. please consolidate them.

Reply:
Done.